Source/WebKit/UIProcess/API/Cocoa/WKWebViewConfigurationInternal.h

```diff
@@ -23,9 +23,12 @@
  * THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#import <WebKit/WKWebViewConfigurationPrivate.h>
+
+#ifdef __cplusplus
```

Comment:
I think it also _is_ possible to consume C++ in Swift modules now, although that might not be unblocked yet.

Reply:
C++ interop is currently disabled in WebKit (I forget exactly why, there's a bug tracking it though)